diffkemp/simpll/simpll.py

```diff
@@ -132,34 +136,53 @@ def run_simpll(first, second, fun_first, fun_second, var, suffix=None,
             if var:
                 simpll_command.extend(["--var", var])
             # Suffix for output files
-            if suffix and output_llvm_ir:
+            if suffix and config.output_llvm_ir:
                 simpll_command.extend(["--suffix", suffix])
             # Cache directory with equal function pairs
             if cache_dir:
                 simpll_command.extend(["--cache-dir", cache_dir])
             # Difference pattern configuration path
-            if pattern_config:
-                simpll_command.extend(["--pattern-config",
-                                       pattern_config.path])
-
-            if control_flow_only:
-                simpll_command.append("--control-flow")
-
-            if output_llvm_ir:
+            if config.custom_pattern_config:
+                simpll_command.extend(["--custom-pattern-config",
+                                       config.custom_pattern_config.path])
+
+            # Builtin pattern configuration
+            if config.builtin_patterns.struct_alignment:
+                simpll_command.append("--struct-alignment")
```

Comment:
Should we do:
```suggestion
            if config.builtin_patterns.struct_alignment:
                simpll_command.append("--struct-alignment")
            else:
                simpll_command.append("--struct-alignment=0
                ")
```
Otherwise, when a pattern is not enabled in `config.builtin_patterns`, the default SimpLL option will apply (which would be "on" in most cases and we don't want that).